templates/new/estimator.py.txt

```diff
@@ -0,0 +1,8 @@
+from surround import Estimator
+
+class Main(Estimator):
```

Comment:
Can we please make this `Baseline`? Also, as a convention, please make sure that the module name maps to the class name and that each module only has a single class.